source/Command.cpp

```diff
@@ -70,17 +70,18 @@ const Command Command::AFTERBURNER(ONE << 19, "Fire afterburner");
 const Command Command::CLOAK(ONE << 20, "Toggle cloaking device");
 const Command Command::MAP(ONE << 21, "View star map");
 const Command Command::INFO(ONE << 22, "View player info");
-const Command Command::FULLSCREEN(ONE << 23, "Toggle fullscreen");
-const Command Command::FASTFORWARD(ONE << 24, "Toggle fast-forward");
-const Command Command::FIGHT(ONE << 25, "Fleet: Fight my target");
-const Command Command::GATHER(ONE << 26, "Fleet: Gather around me");
-const Command Command::HOLD(ONE << 27, "Fleet: Hold position");
-const Command Command::HARVEST(ONE << 28, "Fleet: Harvest flotsam");
-const Command Command::AMMO(ONE << 29, "Fleet: Toggle ammo usage");
-const Command Command::AUTOSTEER(ONE << 30, "Auto steer");
-const Command Command::WAIT(ONE << 31, "");
-const Command Command::STOP(ONE << 32, "");
-const Command Command::SHIFT(ONE << 33, "");
+const Command Command::MESSAGE_LOG(ONE << 23, "View message log");
```

Comment:
why exactly are you inserting this command here?
shifting all other commands by one could cause unnecessary trouble in other PRs

Reply:
I know, but this change isn't particularly hard to resolve when merging, and it helps maintain a logical order of the commands.

Reply:
Checking the blame, this is how we typically add new commands.